packages/opentelemetry-exporter-collector/src/types.ts

```diff
@@ -49,6 +59,99 @@ export namespace opentelemetryProto {
     }
   }
 
+  export namespace metrics.v1 {
+    export interface ExportMetricsServiceRequest {
+      resourceMetrics: opentelemetryProto.metrics.v1.ResourceMetrics[];
+    }
+    export interface Metric {
+      metricDescriptor: opentelemetryProto.metrics.v1.MetricDescriptor;
+      int64DataPoints?: opentelemetryProto.metrics.v1.Int64DataPoint[];
+      doubleDataPoints?: opentelemetryProto.metrics.v1.DoubleDataPoint[];
+      histogramDataPoints?: opentelemetryProto.metrics.v1.HistogramDataPoint[];
+      summaryDataPoints?: opentelemetryProto.metrics.v1.SummaryDataPoint[];
+    }
+
+    export interface Int64DataPoint {
+      labels: opentelemetryProto.common.v1.StringKeyValue[];
+      startTimeUnixNano: number;
+      timeUnixNano: number;
+      value: number;
+    }
+
+    export interface DoubleDataPoint {
+      labels: opentelemetryProto.common.v1.StringKeyValue[];
+      startTimeUnixNano: number;
+      timeUnixNano: number;
+      value: number;
+    }
+
+    export interface HistogramDataPoint {
+      labels: opentelemetryProto.common.v1.StringKeyValue[];
+      startTimeUnixNano: number;
+      timeUnixNano: number;
+      value: number;
+      count: number;
+      sum: number;
+      buckets: opentelemetryProto.metrics.v1.HistogramDataPointBucket;
+      explicitBounds: number[];
+    }
+
+    export interface HistogramDataPointBucket {
+      count: number;
```

Comment:
Do I even need these types? They're not supported in OT yet

Reply:
I think its fine to have them and see what the team says